### PR TITLE
1685 migrate aiven for mysql remote replica

### PR DESCRIPTION
**Pull request by deanholbrough**
# What changed, and why it matters

Migrated the Intercom Help Article [Aiven for MySQL Remote Replica](https://app.intercom.com/a/apps/reo7593q/articles/articles/4948943/show) to https://docs.aiven.io

Added the article to Products > MySQL > HowTo > Create MySQL Remote Replica TOC structure

### CODE REVIEW

**Comment by ftisiot** [comment]
Hi @deanholbrough ,
Thanks for the contribution but the build is failing, can you please check in the `Files Changed` tab what's wrong?

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Create MySQL remote replica
+===========================
+
+The goal of the remote replica is to provide a read-only instance of your managed MySQL service in another geographically diverse region. 
+
+This works as an extra measure to protect your data from the unlikely event that a whole region would go down. It can also improve performance if a read replica is placed closer to your end-users that read from the database.
+
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:7]: Please add section title "Create a remote replica" or similar.

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+Here are the steps to provision remote replica
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:8]: Here are the steps to provision remote replica => Take the following steps to provision a remote replica:

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:9]: Please add a full stop at the end of each step.

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+1. Login into the Aiven console using your credentials
+
+2. Navigate to the MySQL instances for which you wish to create a remote replica
+
+3. Click  the "Create a read replica" button
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:14]: Click  the "Create a read replica" button => `Select **Create a read replica**.`

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+The goal of the remote replica is to provide a read-only instance of your managed MySQL service in another geographically diverse region. 
+
+This works as an extra measure to protect your data from the unlikely event that a whole region would go down. It can also improve performance if a read replica is placed closer to your end-users that read from the database.
+
+Here are the steps to provision remote replica
+
+1. Login into the Aiven console using your credentials
+
+2. Navigate to the MySQL instances for which you wish to create a remote replica
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:12]: instances => services?

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 6]
```diff
@@ -0,0 +1,26 @@
+Create MySQL remote replica
+===========================
+
+The goal of the remote replica is to provide a read-only instance of your managed MySQL service in another geographically diverse region. 
+
+This works as an extra measure to protect your data from the unlikely event that a whole region would go down. It can also improve performance if a read replica is placed closer to your end-users that read from the database.
+
+Here are the steps to provision remote replica
+
+1. Login into the Aiven console using your credentials
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:10]: ``Log in to the `Aiven console <https://console.aiven.io/>`_  using your credentials.``

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+2. Navigate to the MySQL instances for which you wish to create a remote replica
+
+3. Click  the "Create a read replica" button
+
+.. image:: /images/products/mysql/mysql-create-read-only-replica.png
+   :alt: Screenshot of the Create Read-Only Replica 
+
+4. For the remote replica, give your service a name, select the cloud provider and region, and choose a suitable Aiven for MySQL plan.
+
+5. Click "Create"
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:21]: ``Select **Create**.``

File: docs/products/mysql/howto/create-remote-replica.rst [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+
+.. image:: /images/products/mysql/mysql-create-read-only-replica-choose-plan.png
+   :alt: Screenshot of the Create Read-Only Replica 
+
+That's it. You will see the read replica being created and listed as any other Aiven service on the Services page on the console.
```
Comment (wojcik-dorota) [inline docs/products/mysql/howto/create-remote-replica.rst:26]:
```
.. topic:: Result

   You can see the read replica being created and listed as any other Aiven service on the **Services** page in the console.
```

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

**Comment by wojcik-dorota** [comment]
@deanholbrough , hope you dont mind I've created a new PR (https://github.com/aiven/devportal/pull/1814) for this one - including all your content and with my comments already implemented. Hope this way I can save us some time delivering  this article.

**Pull request closed without merging**